manim/utils/docbuild/manim_directive.py

```diff
@@ -231,10 +239,15 @@ def run(self):
             dest_dir.mkdir(parents=True, exist_ok=True)
 
         source_block = [
-            ".. code-block:: python",
+            ".. raw:: html",
             "",
+            "    <pre data-interactive>",
             "    from manim import *\n",
             *("    " + line for line in self.content),
+            "",
+            "    # don't remove below command for run button to work",
+            f"    %manim -qm {clsname}",
```

Comment:
I don't necessarily like that these lines are appended to every example in the documentation, but I'd be wiilling to go with it for an initial version. I'll leave some general comments in the overall review.